examples/src/Examples.js

```diff
@@ -83,10 +85,23 @@ export default class Examples extends Component {
     return null;
   }
 
+  processEvents() {
+    if (eventQueue.length > 0) {
+      let event = eventQueue[0];
```

Comment:
the thing is I only want to remove it from the queue when it is actually done processing it